umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10.clj

```diff
@@ -222,4 +222,11 @@
      (ru/generate-access-urls (:RelatedUrls c))
      (ru/generate-resource-urls (:RelatedUrls c))
      (spatial/spatial-element c)
-     (ru/generate-browse-urls (:RelatedUrls c))]))
+     (ru/generate-browse-urls (:RelatedUrls c))
+     (when-let [direct-dist-info (:DirectDistributionInformation c)]
+       [:DirectDistributionInformation
+         [:Region (:Region direct-dist-info)]
+         (for [prefix-name (:S3BucketAndObjectPrefixNames direct-dist-info)]
+           [:S3BucketAndObjectPrefixName prefix-name])
+         [:S3CredentialsAPIEndpoint (:S3CredentialsAPIEndpoint direct-dist-info)]
+         [:S3CredentialsAPIDocumentationURL (:S3CredentialsAPIDocumentationURL direct-dist-info)]])]))
```

Comment:
This looks exactly the same as the `dif10` mapping. Is it worth having a common function between the two formats? Not sure how likely it is that they would diverge